<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="aiBench"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>521</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="playerBench"/>
    </widget>
    <widget class="QLabel" name="Ai_Battle_Monster">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>168</y>
       <width>169</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Player_Battle_Monster">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>168</y>
       <width>169</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>901</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QScrollArea" name="scroll_area">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>871</width>
       <height>251</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>869</width>
        <height>249</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="consoleLog">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    border: 3px solid rgb(38, 114, 255);
    border-radius: 8px;
    background-color: rgb(230, 230, 230);
    padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Export_Button">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>912</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>카드 내보내기</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="button_area"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
